spec/models/user_spec.rb

```diff
@@ -14,6 +14,13 @@
   end
 end
 
+RSpec.describe 'Confirm username' do
+  it 'should have a username' do
+    user = FactoryGirl.create(:user)
```

Comment:
It looks like you're creating a user factory for each test in this file. Moving it into a let/let! statement will clean up your code